### ISSUE TITLE
Option to save the final tree in a PNG?

### ISSUE THREAD

**Issue by MadeInPierre** [MadeInPierre/finalynx#50]
Could be nice to generate this image daily in a server somewhere (home raspberry pi?) and get a daily/weekly image of our portfolio

**Comment by lasconic** [comment]
Not a PNG, but what about HTML ? 
See https://randomgeekery.org/post/2021/08/extracting-rich-output/

**Comment by MadeInPierre** [comment]
That was the first thing I tried in the web dashboard (see https://github.com/zauberzeug/nicegui/discussions/725) but the output wasn't perfect as there was some display glitches when exporting to HTML. It was pretty decent though for a temporary solution, but I replaced it with the interactive NiceGUI Tree available today.

The best way would be to (optionally) turn Finalynx into a web server, host it on a raspberry pi, create a nice/responsive/mobile web dashboard, and use it from our phone/desktop browsers. However, I'm a noob in web development and can't do all this :sweat_smile: If Finary supports OAuth one day, Finalynx could be hosted somewhere like AWS and be available for everyone.

Otherwise, something like a daily PNG post to our Telegram account could be nice too but a bit more niche. I haven't found a way to generate a PNG as beautiful as the terminal output yet.
